rest/src/main/java/com/redhat/ipaas/rest/v1/handler/connection/ConnectorHandler.java

```diff
@@ -38,4 +39,9 @@ public String resourceKind() {
         return Connector.KIND;
     }
 
+    @Path("/{id}/actions")
+    public ConnectorActionHandler getActions(@PathParam("id") String connectorId) {
```

Comment:
Didn't know that you can return a handler. Again a noob question: From where does JAX RS know that it shouldn't use the handler as response object but look into for further processing ?